holoviews/core/util.py

```diff
@@ -1516,6 +1516,8 @@ def resolve_dependent_value(value):
        resolved.
     """
     range_widget = False
+    if isinstance(value, (tuple, list)):
+        value = [resolve_dependent_value(v) for v in value]
```

Comment:
Seems like this would silently convert tuples to lists, which I assume would cause problems. So maybe:

```suggestion
    if isinstance(value, list):
        value = [resolve_dependent_value(v) for v in value]
    elif isinstance(value, tuple):
        value = tuple([resolve_dependent_value(v) for v in value])
```

Reply:
Also, should there be a case to handle `dict` arguments, presumably recursively calling `resolve_dependent_value` on all the values of the dict? There are pandas methods that accept dictionaries...

Reply:
This is now done. One of the failed test was because of this conversion.